SimplenoteUITests/Generic.swift

```diff
@@ -65,9 +71,14 @@ class Table {
     }
 
     class func trashCell(noteName: String) {
```

Comment:
I'd suggest making the fact that this method not only finds the trash cell for the given note but also action it in the name.

```suggestion
    class func actionTrashCell(noteName: String) {
```

or 

```suggestion
    class func deleteNoteViaTrashCell(noteName: String) {
```